app/Http/Controllers/Models/LinkController.php

```diff
@@ -31,11 +31,14 @@ public function index(Request $request): View
 
         $links = Link::byUser()
             ->with('tags')
-            ->orderBy($orderBy, $orderDir)
-            ->paginate(getPaginationLimit());
+            ->orderBy($orderBy, $orderDir);
```

Comment:
This should be put into an `else` to the statement below. When ordering by random, this would order by the `random` column which does not exists.

Reply:
I was eyeing that line as well, though it works for me (in my local `master`, that's still how it works) so I was now digging into whether I made any other changes, but it doesn't look like it. Putting it in an `else` is probably the most sensible thing to do here. I'll open up a new pull request. (I can't simply append to this one because it's already merged, right?)

Reply:
One difference is probably the database engine: I'm using sqlite. But that also isn't a fan of unknown columns in ordering clauses so it's not an explanation by itself.

```
sqlite> select * from users order by blaat;
Error: no such column: blaat
sqlite> 
```